package.json

```diff
@@ -11,13 +11,15 @@
     "clear": "docusaurus clear",
     "serve": "docusaurus serve",
     "write-translations": "docusaurus write-translations",
-    "write-heading-ids": "docusaurus write-heading-ids"
+    "write-heading-ids": "docusaurus write-heading-ids",
+    "code_snippets": "node src/scripts/generate_code_snippets.mjs"
   },
   "dependencies": {
     "@docusaurus/core": "2.1.0",
     "@docusaurus/preset-classic": "2.1.0",
     "@mdx-js/react": "^1.6.22",
     "clsx": "^1.2.1",
+    "curlconverter": "^4.5.0",
```

Comment:
note: the library which we use to translate the code snippets

Reply:
Just for consistency with the other scripts we should name it `translate-snippets`? WDYT?